homeassistant/components/automation/__init__.py

```diff
@@ -214,6 +215,7 @@ async def reload_service_handler(service_call):
         if conf is None:
             return
         await _async_process_config(hass, conf, component)
+        hass.bus.async_fire(EVENT_AUTOMATION_RELOADED, context=service_call.context)
```

Comment:
Should this be an event or should we just provide this as a trigger? We can hook it in here https://github.com/home-assistant/core/blob/dev/homeassistant/components/automation/homeassistant.py#L41-L46

Reply:
By making is an event, we can hook up to that event in the suggested location?

Reply:
So back to the question should this be an event or trigger:

I guess an event, as we do for others as well. However, this event can be a trigger for the homeassistant automations.

```yaml
trigger:
  - platform: homeassistant
    event: scene_reloaded
```

However, I'm not sure what additional value that would bring. That said, from the UI perspective, we could consider attaching some event triggers (like this one) to make them available via the automation editor in the `homeassistant` trigger type.

Some examples, top of mind, besides these in this PR:

- config entry/device discovered?
- persistent notification updated/created?
- user added/removed/changed?

Reply:
well I am personally a fan of only adding things until we see a clear use case. These are things that we need to support forever, including when we drop the UI+YAML for configuration and just load based on cyberthoughts.

Reply:
Reduced the case, to just scene & automation. Added it as 2 new built-in events. Documentation added as well.